okhttp/src/jvmMain/kotlin/okhttp3/Cookie.kt

```diff
@@ -243,6 +261,8 @@ class Cookie private constructor(
         append("; httponly")
       }
 
+      append("; samesite=").append(sameSite)
```

Comment:
Omit if `"None"`

Reply:
If someone intentionally set the value to `"Lax"`, it seems unexpected to omit it from the cookie string.

Reply:
I think we should make `sameSite` nullable so we can distinguish between explicitly setting `Lax` vs. not